parts/bevezetes.tex

```diff
@@ -0,0 +1,16 @@
+%1. fejezet
+Az okos-telefonok térhódítása miatt már hazánkban is a lakosság fele rendelkezik valamilyen okos eszközzel, ez a szám pedig a jövőben egyre csak növekedni fog. A mindennapi élet megkönnyítésére rengeteg féle alkalmazás születik napról napra. Egyetlen eszközön olvashatunk újságot, tudakozódhatunk a közlekedésről, vehetünk ebédet magunknak, vagy unaloműzésként játszhatunk. Manapság minden korosztály talál kedvére való játékot, legyen szó akár ingyenes akár fizetős verzióról. Napjainkban igencsak elterjedtek az olyan játékok ahol a felhasználók interakciókba léphetnek egymással. Ezek túlnyomó többsége az úgynevezett micro-paymentekre alapszik, ahol a játékosok csekély összegekért cserébe előnyökhöz, könnyítésekhez juthatnak. A „free to play, pay to win” kifejezést azokra a játékokra szokták használni, ahol az előbb említett vásárlások nélkül képtelenség megnyerni a játékot, mert túlzottan befolyásolják a játékosok fejlődését.
```

Comment:
Celszeru mondatonkent entert utni. Kesobb konnyebb megtalalni a pdf alapjan valamit a kodban, es review-olni is egyszerubb